src/js/common/utils/options-for-select.js

```diff
@@ -359,6 +359,7 @@ const states = {
     { label: 'Ohio', value: 'OH' },
     { label: 'Oklahoma', value: 'OK' },
     { label: 'Oregon', value: 'OR' },
+    { label: 'Other', value: 'Other' },
```

Comment:
@jbalboni @knkski if either of you get a chance today, can you tackle this? I'd like to get this into master after the production branch goes out so we can test on staging.